packages/kbn-plugin-helpers/tasks/build/integration_tests/create_build.test.js

```diff
@@ -20,10 +20,10 @@
 const { resolve } = require('path');
 const { readdirSync, existsSync, unlink } = require('fs');
 const del = require('del');
-const createBuild = require('./create_build');
+const createBuild = require('../create_build');
 
-const PLUGIN_FIXTURE = resolve(__dirname, '__fixtures__/create_build_test_plugin');
-const PLUGIN = require('../../lib/plugin_config')(PLUGIN_FIXTURE);
+const PLUGIN_FIXTURE = resolve(__dirname, '../__fixtures__/create_build_test_plugin');
+const PLUGIN = require('../../../lib/plugin_config')(PLUGIN_FIXTURE);
```

Comment:
Flaky test, sometimes takes a little bit more than 5 seconds, so I moved it into the jest integration tests.